src/utils/RemindersContext.tsx

```diff
@@ -44,19 +48,29 @@ const RemindersContext = createContext<RemindersContextType | undefined>(
 export const RemindersProvider: React.FC<{ children: React.ReactNode }> = ({
   children,
 }) => {
-  const [lists, setLists] = useState<List[]>([]);
+  const [lists, setLists] = useState<List[]>([
+    { id: 'Today', name: 'Today', tasks: [] },
+    { id: 'Scheduled', name: 'Scheduled', tasks: [] },
+    { id: 'All', name: 'All', tasks: [] },
+    { id: 'Flagged', name: 'Flagged', tasks: [] },
+  ]);
   const [activeListId, setActiveListId] = useState<string | null>(null);
-  const [bubbleLists, setBubbleLists] = useState<List[]>([]);
+  const [name, setName] = useState<string>('');
```

Comment:
/nit unnecessary explicit type definition `<string>` - can be inferred from initial value, i.e this is the same as writing:
```
const [name, setName] = useState('');
```